app/poros/search_results.rb

```diff
@@ -1,22 +1,30 @@
 class SearchResults
 
-  def repos(token)
-    json = GithubService.new.list_repos(token)
-    @repos = json[0..4].map do |user_data|
+  def initialize(token)
+    @token = token
+  end
+
+  def github_service(token)
+    GithubService.new(@token)
+  end
+
+  def repos
+    data = github_service(@token).list_repos
+    @repos = data[0..4].map do |user_data|
      Repo.new(user_data)
     end
   end
 
-  def followers(token)
-    json = GithubService.new.list_followers(token)
-    @followers = json.map do |user_data|
+  def followers
```

Comment:
This looks much cleaner having the token passed in as an instance variable, nice refactor.